src/connection/sqlJob.ts

```diff
@@ -102,91 +103,61 @@ export class SQLJob {
     return connectResult;
   }
 
-  async query<T>(sql: string, parameters?: (number | string)[]): Promise<T[]> {
-    const hasParms = (parameters && parameters.length > 0);
+  query<T>(isCL, sql: string, parms: any[] = undefined): Query<T> {
```

Comment:
No type on `isCL`

Reply:
I think it should look like this:

`query<T>(query: string, options: QueryOptons): Query<T>`.

`QueryOptions` should have `parms` and `isCL`

Reply:
resolved at https://github.com/halcyon-tech/vscode-db2i/pull/23/commits/59be615be0a6a012724922f76b02eec049112866